.gitignore

```diff
@@ -41,7 +41,5 @@ jspm_packages/
 # built artifacts
 build
 
-# App Config Secrets
-web-app/app.config.js
-
 .DS_Store
+dist/
```

Comment:
remember to add `.env` to `.gitignore` — you'll only use it for development.  On Netlify you'll set the environment variables via their web interface